### DISCUSSION TITLE
Login plugin for 1.19

### DISCUSSION THREAD

**Discussion by MadPanCake** [games647/FastLogin#884] [Q&A] [answered]
Hello,

I know that this question doesn't really have a lot to do with the FastLogin itself but I just can't find an up to date (1.19)
login plugin that will support FastLogin.

May I ask you guys to tell me which Login plugins **do support** FastLogin and work on **1.19**?

**Comment by ApliNi** [comment, marked as answer]
`AuthMeReloaded`, This plugin works fine in 1.19 even if it has not been updated for a long time. 
https://www.spigotmc.org/resources/authmereloaded.6269/
